pkg/systemd/service-details.jsx

```diff
@@ -623,24 +623,20 @@ export class ServiceDetails extends React.Component {
                 />
                 }
                 <CardHeader>
-                    <Flex className="service-top-panel" spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
-                        <CardTitle>
-                            <Text component={TextVariants.h2} className="service-name">{this.props.unit.Description}</Text>
-                            {this.state.isPinned &&
-                                <Tooltip content={_("Pinned unit")}>
-                                    <ThumbtackIcon className='service-thumbtack-icon' />
-                                </Tooltip>}
-                        </CardTitle>
+                    <Flex className="service-top-panel" spaceItems={{ default: 'spaceItemsMd' }} alignItems={{ default: 'alignItemsCenter' }}>
```

Comment:
So the class is now only kept for the selectors in the tests, not for CSS any more, right?